src/sigstore.ts

```diff
@@ -66,30 +78,42 @@ export async function signAttestation(
   options: SignOptions = {}
 ): Promise<Bundle> {
   const fulcio = new Fulcio({ baseURL: options.fulcioBaseURL });
-  const rekor = new Rekor({ baseURL: options.rekorBaseURL });
+  const tlog = newTLogClient(options);
   const idps = configureIdentityProviders(options);
   const signer = new Signer({
     fulcio,
-    rekor,
+    tlog,
     identityProviders: idps,
   });
 
   const bundle = await signer.signAttestation(payload, payloadType);
-  return BundleSerializer.toJSON(bundle) as Bundle;
+  return bundleToJSON(bundle) as Bundle;
 }
 
 export async function verify(
   bundle: Bundle,
   data?: Buffer,
   options: VerifierOptions = {}
 ): Promise<boolean> {
-  const rekor = new Rekor({ baseURL: options.rekorBaseURL });
-  const verifier = new Verifier({ rekor });
+  const tlog = newTLogClient(options);
+  const verifier = new Verifier({ tlog });
 
-  const b = BundleSerializer.fromJSON(bundle);
+  const b = bundleFromJSON(bundle);
   return verifier.verify(b, data);
 }
 
+// Accepts a signed DSSE envelope and a PEM-encoded public key to be added to the
+// transparency log. Returns a Sigstore bundle suitable for offline verification.
+export async function addEnvelopeToTransparencyLog(
+  envelope: Envelope,
+  key: string,
```

Comment:
Any reason we wouldn't name these `dsseEnvelope` and `publicKey` to match the stuff in bunde/rekor?
```suggestion
  dsseEnvelope: Envelope,
  publicKey: string,
```

Reply:
👍